File: 0070 - Climbing Stairs.py
class Solution:
    def climbStairs(self, n: int) -> int:
        if (n <= 1):
            return 1
        else:
            l = [1, 1]
            for i in range(n - 1):
                l.append(l[i] + l[i + 1])
            return(l[n])

File: 0070 - Climbing Stairs.c
int climbStairs(int n)
{
    if (n <= 1)
    {
        return 1;
    }
    else
    {
        int ways[n + 1];
        ways[0] = 1;
        ways[1] = 1;
        for (int i = 2; i < n + 1; i++)
        {
            ways[i] = ways[i - 2] + ways[i - 1];
        }
        return ways[n];
    }
    
}